README.md

```diff
@@ -1 +1,184 @@
-# recat-todo-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>🛠 React Todo App</b></b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖React Todo App ](#-react-todo-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# React Todo App <a name="about-project"></a>
+
+The react todo app was an introduction to concepts like but not limited to props, state,lifecycle, hooks, routing, nested routing and events. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>REACT</summary>
+  <ul>
+    <li><a href="https://create-react-app.dev/">REACT</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Styling with Vanilla CSS
+- Linting with Eslint and Stylelint.
+- Persisting data with localStorage
+- React Router
+- useState
+- useEffect
+- useRef
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming soon
```

Comment:
Optional: 
♻️  You can improve your project for others, including one deploy link for a faster review in case a recruiter views your work at a glance.